server/routes/index.ts

```diff
@@ -33,6 +34,8 @@ export default function routes(services: Services): Router {
   updateInduction(router, services)
   postInductionCreation(router, services)
 
+  router.use('/plan/:prisonNumber/notes', notesRoutes())
```

Comment:
Re: this line, and the next file .... is this the preferred way of setting up routes in express?
Assuming this is the preferred way we can start to define routes using this pattern and I'll create a tech-debt ticket to refactor the other routes to use this pattern as well

Reply:
I guess "preferred" might depend on who you ask, but I think generally, yes, because it's more a flexible way to assign routes.

Doing it this way would allow us to define the notes routes independently and relative to whatever route the notes router has been assigned to which would make them easier to manage and move as a group. We could even assign the notes router to multiple parent routes if we wanted.